backend/main.py

```
@@ -302,82 +330,58 @@ def create_spot(
+def delete_spot(
+    spot_id: UUID,
+    current_user: Annotated[schemas.AuthorInfo, Depends(get_current_user)],
+    db: Session = Depends(get_db)
+):
+    """
+    スポットを削除（作成者のみ）。
+    """
+    try:
+        crud.delete_spot(db, spot_id, current_user.id)
+    except ValueError:
+        raise HTTPException(status_code=404, detail="Spot not found")
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)